### ISSUE TITLE
`now alias domain` no longer works in 16.2

### ISSUE THREAD

**Issue by bdougherty** [vercel/vercel#3079]
With previous releases I could do `now alias domain.com` and the latest deployment would be aliased to that domain name, but with 16.2 this no longer works and gives the following error:

```
Error! The provided argument "undefined" is not a valid domain
```

I am running the following command in a GitHub action:

```
now alias --token="${NOW_TOKEN}" "${ALIAS}"
```

**Comment by doowb** [comment]
I am also seeing this in a GitHub action when using a `rules.json` file.

**Comment by mcsdevv** [comment]
This has been resolved in the 18.0.0 release.
